sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided \
+atomic memory update operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops/"
```

Comment:
```diff
-HOMEPAGE="https://github.com/ivmai/libatomic_ops/"
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
```

BTW, in `DESCRIPTION`, you might wish to move the words `atomic memory update` (currently in the beginning of line 3) to the end of line 2.
(This would be more consistent with our recommended "80-chars limit" _philosophy_...)